AGGREGATE.md

```diff
@@ -150,12 +142,12 @@ The scheme above will generate the following abstract histogram contributions:
 [
 // campaignCounts
 {
-  key: 1382, // 0b10101100110
+  key: 0x566, // 0b10101100110
   value: 32768 
 },
 // geoValue:
 {
-  key: 181, // 0b000010110101
+  key: 0xB5, // 0b000010110101
```

Comment:
source_key: 0x5
trigger_key: 0xA80
final_key: 0xA85

Reply:
Done (this one should have been obvious to me... :) )